File: ClassForgeAPI/src/main/java/com/classforge/api/ClassForgeAPI/dao/ContactInfo.java
package com.classforge.api.ClassForgeAPI.dao;

import jakarta.persistence.*;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
@Table(name = "contact_info")
public class ContactInfo {
    @Id
    @GeneratedValue(strategy = jakarta.persistence.GenerationType.IDENTITY)
    private Integer id;

    @Column(name="type")
    private String type;

    @Column(name="value")
    private String value;

    @ManyToOne
    @JoinColumn(name="user_id")
    private User user;


}

File: ClassForgeWeb/ClassForgeWeb/src/app/shared/ContactInfo.ts
export class ContactInfo{
  id: number;
  type: string;
  value: string;
  user_id: number;
  constructor(data: any  ){
    this.id=data.id;
    this.type=data.type;
    this.value=data.value;
    this.user_id=data.user_id;

  }
}
